changelog/18.0/18.0.0/summary.md

```diff
@@ -118,6 +119,35 @@ VTGate flag:
 
 - `--schema_change_signal_user` is deprecated and will be removed in `v19.0`
 
+#### <a id="deleted-flags"/>Deleted Command Line Flags
+
+Flags in `vtcombo`:
+- `--vtctld_addr`
+
+Flags in `vtctldclient ApplySchema`:
+- `--skip-preflight`
+
+Flags in `vtgate`:
+- `--vtctld_addr`
+
+Flags in `vttablet`:
+- `--vtctld_addr`
+- `--use_super_read_only`
+- `--disable-replication-manager`
+- `--init_populate_metadata`
+- `--queryserver-config-pool-prefill-parallelism`
+- `--queryserver-config-stream-pool-prefill-parallelism`
+- `--queryserver-config-transaction-pool-prefill-parallelism`
```

Comment:
I don't see this one ever explicitly mentioned in previous changelogs but I'm assuming it was an oversight and should be deleted because all of the related ones were deprecated.